sim/midas/src/main/cc/Makefile

```diff
@@ -7,39 +7,39 @@ r_dir = $(abspath ../resources)
 ########################################################################
 # Parameters:
 # 1) PLATFORM: FPGA platform board(by default zynq)
-# 2) DESIGN: Target design of midas
+# 2) DRIVER_NAME: Base name for compiled drivers and ML simulators
```

Comment:
There's nothing in this makefile that actually needs to know the _name_ of the top-level class. It's just used to provide a name to things in a variety of places. So really, it's just the name of the target for recipes that also happens to be name of the compiled drivers and simulators. 

I want to get signoff of metasimulator before i commit to it. I mostly dropped it in this PR to try to fish for comments.

Reply:
Oh, yeah, would help if I looked at the context before writing my comment.  This is just in the driver Makefile and the toplevel makefile still uses DESIGN and that does need to be the toplevel classname because it's what is used in the generator invocation?  Gotcha.  This makes sense.